.gitignore

```diff
@@ -3,6 +3,7 @@
 *.exe~
 *.dll
 *.so
+*.log
```

Comment:
Is this file generated by the `ie`? I haven't noticed it.

Reply:
Yeah, for me. I am getting ie.log once running it. Maybe it is only written if something fails?

Reply:
I don't know. Please keep track of this things so that we can provide feedback to the InnovationEngine project.